templates/experiments/single_experiment_home.html

```diff
@@ -389,4 +504,139 @@ <h3 class="font-bold text-lg">Parent Experiments:
 
     window.addEventListener('hashchange', selectTabFromHash);
   </script>
+  <script>
+    const tags = {{ available_tags|safe|default:"[]" }};
+    const versionsList = {{ experiment_versions|safe|default:"[]" }};
```

Comment:
It would be safer to pass these using the [json_script](https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#std-templatefilter-json_script) filter